include/dfrn.php

```diff
@@ -1555,11 +1687,13 @@ private function process_suggestion($xpath, $suggestion, $importer) {
 	 * @param object $xpath XPath object
 	 * @param object $relocation relocation elements
 	 * @param array $importer Record of the importer user mixed with contact of the content
+	 * @todo Find good type-hints for all parameter
 	 */
 	private function process_relocation($xpath, $relocation, $importer) {
 
 		logger("Processing relocations");
 
+		/// @TODO Rewrite this to one statement
```

Comment:
It makes code be easier and is a common rule:
````
if ($foo == 'for') {
if ($bar != 'bar') {
doFooNoBar();
}
}
````
... can be rewritten to:
````
if ($foo == 'for' && $bar != 'bar') {
doFooNoBar();
}
````
... is much smaller and lesser nested curly braces.